### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by DominicMaas**
Keeping this one simple, I simply ran a code formatter (CodeMaid) across the entire code base, and updated any outdated or broken links.

Also removed the deprecated logging method as nothing uses it anymore.

Lots of changes, but are quite small (whitespace mainly), trying to keep everything clean.

### CODE REVIEW

**Comment by DominicMaas** [comment]
Sorry about this one :( 

Quite a lot to go through!

Hopefully we only ever need to do this very rarely.

File: src/Commands/Handler/Internal/ClientJoiningHandler.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 namespace CSM.Commands.Handler.Internal
 {
-    class ClientJoiningHandler : CommandHandler<ClientJoiningCommand>
+    internal class ClientJoiningHandler : CommandHandler<ClientJoiningCommand>
```
Comment (kaenganxt) [inline src/Commands/Handler/Internal/ClientJoiningHandler.cs]: Is there a reason why this class is marked as internal?

Comment (DominicMaas) [reply src/Commands/Handler/Internal/ClientJoiningHandler.cs]: Ahh, missed that one. Should be public. Will fix this afternoon.

**Review by kaenganxt — APPROVED** [review]
LGTM

**Pull request merged**